### PR TITLE
Add recipe for universal-ctags

### PR DESCRIPTION
**Pull request by xhochy**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge/staged-recipes This is ready for review/merge.

File: recipes/universal-ctags/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+./autogen.sh
+./configure --prefix=$PREFIX
+make -j${CPU_COUNT}
+make install
```
Comment (ocefpaf) [inline recipes/universal-ctags/build.sh:4]: I'm assuming there is not a make check or similar, right?

Comment (xhochy) [reply recipes/universal-ctags/build.sh:4]: Maybe, I'll have a look on the feedstock

File: recipes/universal-ctags/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - automake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkg-config
+    - make
+  host:
+    - libiconv
+    - libxml2
+    - jansson
+    - libseccomp  # [linux]
```
Comment (mbargull) [inline recipes/universal-ctags/meta.yaml:28]: Why?

Comment (mbargull) [reply recipes/universal-ctags/meta.yaml:28]: Meaning, why not on Linux?

Comment (xhochy) [reply recipes/universal-ctags/meta.yaml:28]: This is marked as Linux-only as it's a Linux-only package.

Comment (xhochy) [reply recipes/universal-ctags/meta.yaml:28]: @mbargull Are you reading the condition in the wrong way?

Comment (mbargull) [reply recipes/universal-ctags/meta.yaml:28]: What do you mean? I've never said anything ;). Bye!

Comment (mbargull) [reply recipes/universal-ctags/meta.yaml:28]: (Yep, sorry for the noise!)

**Pull request merged**